runtime/src/lib.rs

```diff
@@ -861,12 +865,26 @@ construct_runtime!(
         UncheckedExtrinsic = UncheckedExtrinsic
     {
         System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Call, Storage},
         Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
-        //Indices: pallet_indices::{Pallet, Call, Storage, Event<T>, Config<T>},
-        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-        TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
         Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
+        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage},
+        TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
+
+        // PARACHAIN
+		ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned},
```

Comment:
here they have `= 20` at end of this line  https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L460, maybe due to newer version

Reply:
Done